diff -uNr openbts-P2.8.0Opelousas/apps/Makefile.in openbts-P2.8.0Opelousas.new/apps/Makefile.in
--- openbts-P2.8.0Opelousas/apps/Makefile.in	2014-01-05 21:22:19.563299794 +0800
+++ openbts-P2.8.0Opelousas.new/apps/Makefile.in	2014-01-05 21:35:14.933279293 +0800
@@ -372,7 +372,7 @@
 GLOBALS_LA = $(top_builddir)/Globals/libglobals.la
 CLI_LA = $(top_builddir)/CLI/libcli.la
 SR_LA = $(top_builddir)/SR/libSR.la
-SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la
+SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la -ldl
 MOSTLYCLEANFILES = *~
 AM_CPPFLAGS = $(STD_DEFINES_AND_INCLUDES)
 AM_CXXFLAGS = -Wall -ldl -pthread
diff -uNr openbts-P2.8.0Opelousas/apps/OpenBTS.cpp openbts-P2.8.0Opelousas.new/apps/OpenBTS.cpp
--- openbts-P2.8.0Opelousas/apps/OpenBTS.cpp	2011-10-12 05:58:01.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/apps/OpenBTS.cpp	2014-01-05 21:25:45.759961009 +0800
@@ -138,7 +138,7 @@
 
 	gConfig.setUpdateHook(purgeConfig);
 	gLogInit("openbts",gConfig.getStr("Log.Level").c_str(),LOG_LOCAL7);
-	LOG(ALERT) << "OpenBTS starting, ver " << VERSION << " build date " << __DATE__;
+	//LOG(ALERT) << "OpenBTS starting, ver " << VERSION << " build date " << __DATE__;
 
 	COUT("\n\n" << gOpenBTSWelcome << "\n");
 	COUT("\nStarting the system...");
diff -uNr openbts-P2.8.0Opelousas/CommonLibs/Makefile.in openbts-P2.8.0Opelousas.new/CommonLibs/Makefile.in
--- openbts-P2.8.0Opelousas/CommonLibs/Makefile.in	2014-01-05 21:22:19.063299807 +0800
+++ openbts-P2.8.0Opelousas.new/CommonLibs/Makefile.in	2014-01-05 21:29:32.459955014 +0800
@@ -418,7 +418,7 @@
 GLOBALS_LA = $(top_builddir)/Globals/libglobals.la
 CLI_LA = $(top_builddir)/CLI/libcli.la
 SR_LA = $(top_builddir)/SR/libSR.la
-SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la
+SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la -ldl
 MOSTLYCLEANFILES = *~ testSource testDestination
 AM_CPPFLAGS = $(STD_DEFINES_AND_INCLUDES)
 AM_CXXFLAGS = -Wall -ldl -O3 -g -lpthread
diff -uNr openbts-P2.8.0Opelousas/CommonLibs/SocketsTest.cpp openbts-P2.8.0Opelousas.new/CommonLibs/SocketsTest.cpp
--- openbts-P2.8.0Opelousas/CommonLibs/SocketsTest.cpp	2011-10-12 05:58:02.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/CommonLibs/SocketsTest.cpp	2014-01-05 21:50:54.429921120 +0800
@@ -30,6 +30,7 @@
 #include "Threads.h"
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 
 static const int gNumToSend = 10;
diff -uNr openbts-P2.8.0Opelousas/CommonLibs/Timeval.h openbts-P2.8.0Opelousas.new/CommonLibs/Timeval.h
--- openbts-P2.8.0Opelousas/CommonLibs/Timeval.h	2011-10-12 09:20:05.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/CommonLibs/Timeval.h	2014-01-05 21:24:15.336630067 +0800
@@ -29,6 +29,7 @@
 #include <stdint.h>
 #include "sys/time.h"
 #include <iostream>
+#include <unistd.h>
 
 
 
diff -uNr openbts-P2.8.0Opelousas/SIP/SIPEngine.cpp openbts-P2.8.0Opelousas.new/SIP/SIPEngine.cpp
--- openbts-P2.8.0Opelousas/SIP/SIPEngine.cpp	2011-10-12 09:19:23.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/SIP/SIPEngine.cpp	2014-01-05 21:25:19.576628369 +0800
@@ -818,7 +818,7 @@
 	get_rtp_params(msg, d_port, d_ip_addr);
 	LOG(DEBUG) << "IP="<<d_ip_addr<<" "<<d_port<<" "<<mRTPPort;
 
-	rtp_session_set_local_addr(mSession, "0.0.0.0", mRTPPort );
+	rtp_session_set_local_addr(mSession, "0.0.0.0", mRTPPort, -1 );
 	rtp_session_set_remote_addr(mSession, d_ip_addr, atoi(d_port));
 
 	// Check for event support.
diff -uNr openbts-P2.8.0Opelousas/TransceiverRAD1/DummyLoad.cpp openbts-P2.8.0Opelousas.new/TransceiverRAD1/DummyLoad.cpp
--- openbts-P2.8.0Opelousas/TransceiverRAD1/DummyLoad.cpp	2011-10-12 05:58:04.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/TransceiverRAD1/DummyLoad.cpp	2014-01-05 21:23:23.359964774 +0800
@@ -33,6 +33,7 @@
 #include <stdint.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 #include "Threads.h"
 #include "DummyLoad.h"
 
diff -uNr openbts-P2.8.0Opelousas/TransceiverRAD1/Makefile.in openbts-P2.8.0Opelousas.new/TransceiverRAD1/Makefile.in
--- openbts-P2.8.0Opelousas/TransceiverRAD1/Makefile.in	2014-01-05 21:22:19.516633130 +0800
+++ openbts-P2.8.0Opelousas.new/TransceiverRAD1/Makefile.in	2014-01-05 21:30:16.229953858 +0800
@@ -404,7 +404,7 @@
 GLOBALS_LA = $(top_builddir)/Globals/libglobals.la
 CLI_LA = $(top_builddir)/CLI/libcli.la
 SR_LA = $(top_builddir)/SR/libSR.la
-SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la
+SQLITE_LA = $(top_builddir)/sqlite3/libsqlite.la -ldl
 MOSTLYCLEANFILES = *~
 AM_CPPFLAGS = $(STD_DEFINES_AND_INCLUDES) $(USB_INCLUDES) $(WITH_INCLUDES)
 AM_CXXFLAGS = -DOMNITHREAD_POSIX=1 -lusb-1.0 -ldl -lpthread
diff -uNr openbts-P2.8.0Opelousas/TransceiverRAD1/rnrad1Core.h openbts-P2.8.0Opelousas.new/TransceiverRAD1/rnrad1Core.h
--- openbts-P2.8.0Opelousas/TransceiverRAD1/rnrad1Core.h	2011-10-12 09:20:48.000000000 +0800
+++ openbts-P2.8.0Opelousas.new/TransceiverRAD1/rnrad1Core.h	2014-01-05 21:55:40.549913554 +0800
@@ -40,6 +40,7 @@
 #include <string.h>
 #include <cstdio>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include "Logger.h"
 
